<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chart</class>
 <widget class="QMainWindow" name="chart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>收入</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_8">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="8">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="cbx_IncomeTime_2">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QDoubleSpinBox" name="spinPieSize">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>0.700000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>饼图大小</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cbx_incomeTypeOfTime_2">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>选择时间</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QDoubleSpinBox" name="spinHoleSize">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="pbtn_drawIncomePieChart_2">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #e67e22; /* 橘色背景 */
    color: white; /* 白色文字 */
    border: 1px solid #e67e22; /* 橘色边框 */
    border-radius: 5px; /* 圆角 */
    padding: 5px 10px; /* 内边距 */
    font-size: 14px; /* 文字大小 */
}

QPushButton:hover {
    background-color: #d35400; /* 鼠标悬停时的颜色 */
}

QPushButton:pressed {
    background-color: #a04000; /* 鼠标按下时的颜色 */
}
</string>
             </property>
             <property name="text">
              <string>绘制饼状图</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>孔径尺寸</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QChartView" name="IncomeGraphicsView_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>支出</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_9">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="4">
            <widget class="QPushButton" name="pbtn_drawExpendPieChart_2">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #e67e22; /* 橘色背景 */
    color: white; /* 白色文字 */
    border: 1px solid #e67e22; /* 橘色边框 */
    border-radius: 5px; /* 圆角 */
    padding: 5px 10px; /* 内边距 */
    font-size: 14px; /* 文字大小 */
}

QPushButton:hover {
    background-color: #d35400; /* 鼠标悬停时的颜色 */
}

QPushButton:pressed {
    background-color: #a04000; /* 鼠标按下时的颜色 */
}
</string>
             </property>
             <property name="text">
              <string>绘制饼状图</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="cbx_ExpendTime_3">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>饼图大小</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QDoubleSpinBox" name="spinHoleSize_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>选择时间</string>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QDoubleSpinBox" name="spinPieSize_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>0.700000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cbx_expendTypeOfTime_3">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>孔径尺寸</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QChartView" name="ExpendGraphicsView_3"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
